src/MovieApi.js

```diff
@@ -17,12 +20,14 @@ const MovieApi = (props) => {
                 setMovieKeyword(res.data.keywords);
                 console.log(res.data.keywords);
             })
+            // reset the state of userInput and movieID to an empty string
             setMovieKeyword([])
         }
 }, [props.movieId])
 
     return (
-        movieKeyword.slice(0,3).map((keyword)=>{
+        // passing only 3 keywords randomly to GiphyApi component
+        movieKeyword.sort(() => 0.5 - Math.random()).slice(0,3).map((keyword)=>{
```

Comment:
Good job! Thanks for doing this. I would love to hear you explain sort and why it's 0.5. I need help understanding it better